Makefile

```diff
@@ -111,3 +112,31 @@ codegen:
 .PHONY: clean
 clean:
 	rm -rf $(shell pwd)/_output
+	
+.PHONY: clean-dev
+clean-dev:
+	rm -rf $(shell pwd)/_output/dev
+
+# Use make start-dev when you want to create a kind cluster for
+# testing the authenticator.  You must pass in the admin ARN and
+# the image under test.
```

Comment:
See note below. Consider defaulting the image to one built from the locally checked-out source.